README.md

```diff
@@ -29,6 +29,8 @@ A brief introduction into the tool and its use cases is presented [here](https:/
 
 # Console
 
+Only Python 3.11+ versions are supported for CLI.
```

Comment:
I'm not sure, but probably this should be mentioned in the installation section.